src/routes/game/components/dialogs/components/RulesDialog.vue

```diff
@@ -2,12 +2,17 @@
   <BaseDialog id="rulesDialog" max-width="1250">
     <template #title>
       <h1>Rules of Cuttle</h1>
-      <v-btn icon aria-label="Close rules dialog" @click="close">
+      <v-btn
+        icon 
+        aria-label="Close rules dialog"
+        color="surface-1" 
```

Comment:
Rather than setting the color prop to the same color as the background, let's set the `variant` prop to "text" in order to set the button style to not have a background at all.
```suggestion
        color="surface-2"
        variant="text"
```


I believe this should make it so the icon appears the "cream" color but without a background

Reply:
Sure, so let me paste screenshots of all the close buttons I found to be sure all is good

Reply:
On src/routes/game/components/MoveChoiceOverlay.vue

that button was set to have a white color property, I believe it should be cream too

![Screenshot from 2024-02-11 14-51-54](https://github.com/cuttle-cards/cuttle/assets/7325003/c00e1a06-078f-4aa9-927a-79b3e8711607)

Reply:
On the src/components/BaseSnackbar.vue

No change needed , the background color is already transparent

![Screenshot from 2024-02-11 14-49-30](https://github.com/cuttle-cards/cuttle/assets/7325003/56d2b728-0a5f-453b-a915-05304a2c5de4)

Reply:
On src/routes/game/components/TargetSelectionOverlay.vue

white background, to keep contrast as you mentionned

![Screenshot from 2024-02-11 12-56-41](https://github.com/cuttle-cards/cuttle/assets/7325003/a56da6f9-b128-48c4-ac3f-5953c92e88fb)

Reply:
On src/routes/game/components/dialogs/components/ScrapDialog.vue

Brown background (just like RulesDialog.vue) so I add variant="text" and remove elevation="0"

![Screenshot from 2024-02-11 11-44-00](https://github.com/cuttle-cards/cuttle/assets/7325003/f8f95436-bc56-4814-b2d6-6a49cba70a47)

Reply:
and the src/routes/game/components/dialogs/components/RulesDialog.vue

Which is now cream and without background
![Screenshot from 2024-02-11 11-42-39](https://github.com/cuttle-cards/cuttle/assets/7325003/8d53bd37-4ca2-4750-87aa-5d6d4d483c21)